en/micro-integrator/docs/administer-and-observe/working-with-management-api.md

```diff
@@ -587,3 +587,64 @@ The management API has multiple resources to provide information regarding the d
 	```bash
 	curl -X GET "https://localhost:9164/management/data-services?dataServiceName=StudentDataService" -H "accept: application/json" -H "Authorization: Bearer TOKEN" -k -i
 	```
+	!!! Info
+        **The following logging resource feature is supported in WSO2 MI 1.1.0 as a product update. This update is availble through WSO2 Update Manager from 10th of June, 2020 onwards.**
```

Comment:
The following logging resource is available as a product update (available from **10th June, 2020** onwards). See the instructions on [getting product updates](https://docs.wso2.com/display/updates/Getting+Started).